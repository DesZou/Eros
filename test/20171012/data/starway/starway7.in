158473 861698 1000
123078 189338
101719 107431
38420 153187
94499 619359
110084 810394
113454 239968
132022 602838
76297 167810
8988 345322
13782 532377
83915 113707
26665 131405
71399 842548
102704 21368
39356 383429
63695 765223
145812 741512
147440 220004
42292 554743
88485 472852
150629 30720
105248 523565
90311 61186
11609 302142
86904 80387
144743 432325
11295 750892
88378 477766
134169 66698
47900 142812
137098 287070
9167 676281
150916 363683
78410 847551
93295 367250
38888 257957
26277 285360
60496 844474
76934 618810
130857 30393
28009 227411
133918 834334
102174 59441
10488 465703
152091 377805
146977 69709
139537 503314
66517 73544
104701 89203
3372 236971
23680 308932
133012 34705
3752 25634
52353 37117
86528 828003
18979 418290
70292 394177
121437 237560
110863 572999
122039 434980
92868 460089
112188 412345
143592 231250
16151 521661
2341 18693
129638 211539
80357 46207
47093 576850
38752 152737
76908 742773
102632 402984
51017 155319
4931 313510
21854 306120
17622 279068
31188 159919
65415 846101
55970 275902
150364 821865
6729 735196
92540 432854
56798 104493
55633 522955
86047 719925
73139 319598
90248 205779
61076 82415
156238 103735
83389 822782
15861 165225
47961 514980
121687 563422
28476 341669
89423 428813
46919 480876
626 112532
156528 114070
149054 215899
145259 718002
97971 523110
65208 162309
115364 378868
70043 339898
42618 648275
74684 82943
135564 712639
113369 465296
125835 815777
64087 640224
155550 236591
86254 854867
53337 699357
140724 340011
9388 790314
62998 642306
139862 35109
1255 18459
10575 579848
112741 699757
90742 364041
74007 487450
99554 339039
112723 437082
122271 772711
23895 566265
60269 308941
29444 741434
90345 24098
23399 653696
96168 175565
96836 579368
3934 827189
122132 716537
32089 229350
20730 398597
74102 124632
98998 507514
30849 166475
57228 370047
99356 579404
147568 527520
90065 243819
4529 117751
12032 408030
54368 844608
66909 820382
56583 754884
133131 703834
51018 689390
104075 263364
11434 248591
106636 812423
127424 390994
93977 57098
65067 431615
125505 761996
62245 136132
29268 538222
63962 504402
22026 316880
136219 593645
18413 179704
66062 815068
133460 833271
64591 62826
130094 640873
42383 140106
135740 535487
16230 587829
14851 173375
115112 699237
72256 689339
77012 777950
29581 728274
107738 789053
80165 147981
152653 631483
146622 103879
91039 741842
17324 184312
90492 715307
68554 113342
136564 680580
15146 782038
48284 641600
50195 641922
2204 167513
52337 246086
51194 139420
7932 632869
117561 594442
110236 757213
65917 615664
20672 678111
59479 136550
89719 221650
114089 715647
85888 696877
140761 713131
138029 102837
54715 446028
25278 405605
26031 324772
61374 34803
17588 402896
52417 129068
15227 256923
104207 615009
123208 488415
54279 327242
51070 150429
18148 377844
109425 603080
51970 146920
77654 735063
45162 40265
9263 33542
101395 429663
83651 289146
110656 138998
89965 422992
99768 21339
106868 738415
33293 302500
41948 398712
26234 411253
10405 748362
84170 65588
129262 726298
9974 261764
117073 515615
101918 47773
44906 70365
3283 576735
100442 523431
124494 374928
72271 431465
63002 543247
17506 2353
90811 822691
93076 548074
125747 583173
83472 265792
79811 147858
33570 725861
102418 764835
11663 615696
99553 387711
34016 770764
53286 692856
103354 339638
62930 137372
111015 390076
131910 528417
157295 390177
17583 122164
22603 206083
13132 97197
113136 640465
77633 21511
7019 152716
65585 747264
154475 176886
35329 641320
93044 856601
65860 331843
126346 741907
140142 293615
65559 556868
133319 98197
130053 4644
93913 34663
141936 258289
150613 781303
32480 524298
137105 121199
122473 335828
7364 771986
153245 576516
30121 21239
74760 148949
27995 478652
42845 462484
141207 314561
49058 398856
112033 576019
4125 581864
69678 654927
136404 381548
69536 103295
86089 304416
78032 407307
31410 249832
131298 646382
149703 458854
146106 691784
133585 49441
121865 577798
68340 115123
65408 501762
40746 413787
154634 716610
110319 19633
64056 708174
32299 766330
55052 2789
122220 747776
52232 681809
65870 140138
101855 785658
154815 498700
140730 701063
15226 631202
80441 21142
114224 11805
127745 390643
38220 595213
33359 428826
37263 349972
56394 605462
146358 672090
41372 824821
25820 823403
143465 398569
95593 857545
154757 652072
109924 50321
83666 401765
92465 583271
59987 383007
112428 324972
77336 483521
139785 609638
98982 193699
112257 284475
71101 99724
148002 586626
122354 235393
132593 587468
58296 179674
126951 432824
80862 249078
47494 350635
57495 554884
114516 476061
52366 520944
1994 366086
182 220008
51046 581194
156603 46972
36717 546201
8326 355749
151705 644691
41543 845163
77105 354280
110163 527883
27647 616699
66262 462779
154186 346369
39080 512772
97241 108441
16855 204881
129059 11735
20141 549720
34599 575491
119978 846302
51188 206194
114445 695578
32929 457232
75897 832985
10198 770648
41283 827277
27254 110851
56178 409931
92730 693656
85361 540642
115414 578398
11945 49598
108474 614535
72382 127471
24805 585843
37691 565261
64078 318994
85858 446753
91160 764590
158177 213633
17005 280133
83765 25710
139551 666609
32211 347200
24646 790714
44826 705797
111164 158193
104583 707664
69393 521208
45795 821507
53278 839903
48386 63071
146895 777555
12118 595551
26412 161098
91160 786617
68184 271192
31567 438403
130533 581939
114747 714014
119970 760676
38335 500378
144253 260796
2312 409092
84127 476053
151274 124996
5468 185731
88272 44049
66044 211237
97559 199314
61246 34142
32682 400427
129926 482765
18948 176234
91721 520158
88917 748194
77653 33197
76619 157413
147952 860017
13917 301789
138988 751419
2799 702564
71515 667859
61348 91027
13537 746663
59254 292267
67797 587545
152414 451244
126952 506906
39689 35351
106543 290538
7476 461865
7923 328763
26863 318493
137077 632281
137497 632885
125091 460207
139488 372365
93109 489942
18350 398812
132197 776410
92876 437957
48575 149724
78677 167560
56839 769118
104716 607586
93129 844562
117967 548749
54342 218118
118981 208247
47250 616540
121963 827782
19338 333032
53590 60858
137306 585000
40135 79809
141344 474298
146722 55441
93222 297283
57705 653099
89206 567655
90036 404327
101080 405102
59585 219810
13833 639358
37116 450783
28434 427635
120775 479365
34953 39513
88913 690670
73586 57005
79722 162092
15428 717347
89605 116400
126116 479777
148889 735370
109878 127036
105947 259936
20203 302001
50765 218064
126691 177908
125498 434093
90820 375139
25179 56057
93013 477079
148954 200937
92073 167335
38839 792675
58070 98628
99868 589811
153871 294169
150440 326596
118984 761007
131949 673883
79813 724759
20034 532747
125372 855028
67818 215904
11466 114066
88804 784161
101974 647189
26810 545307
132481 28589
48885 21495
126223 346600
89527 182460
6962 4220
102973 591384
138074 632854
82185 111316
29239 450035
7333 168226
3295 246266
73547 633119
8517 840371
142366 526042
41842 485532
80629 514270
117819 337389
139589 34675
106609 833933
151694 467872
70796 242480
71403 112623
104024 252236
60542 489451
59102 607043
92297 691059
128307 561329
78632 823424
98053 342629
90612 210006
52695 334445
46044 797969
58032 661174
70543 806729
16756 240739
1077 517287
6624 123325
42576 105080
28642 588355
113714 508530
121683 341337
43042 327045
16728 357014
37610 2424
149883 429311
126543 688303
61920 361413
110116 64795
38740 397716
138671 459000
14392 253604
82197 612147
105428 70320
92515 568522
75624 458646
10267 407373
151534 24744
48872 763347
157205 392146
81141 483156
16096 126903
137783 442987
101120 409542
49769 237462
937 47940
82554 772500
5936 372352
76964 458276
154043 122993
56712 575639
72351 843253
73529 691088
106746 244560
14256 450772
56218 837177
113759 226023
105300 57604
43456 78003
147359 698069
59319 523341
87753 858410
56808 680701
81477 319368
121744 225109
15984 686535
19428 409320
62880 540906
156455 687523
114976 683160
136125 555530
154002 422096
15918 246592
154587 500490
81543 783149
42251 655207
3390 350992
31419 90894
17009 150870
24660 525976
70428 632395
49890 693828
44831 534460
108350 53133
61447 696024
121431 459606
155669 465356
80824 339704
21166 357127
70049 342487
104219 754479
72510 210628
45488 769185
44233 485307
56654 582421
145448 860535
147679 14245
86882 667688
138469 562094
152009 399629
62516 717399
37963 92996
76510 115674
10763 682341
84124 439733
53360 172145
147441 190923
50160 424340
1024 827409
28940 267667
115076 718503
124613 596201
74413 69983
114451 501368
9451 385108
145822 357989
21802 491185
65834 761174
51611 504478
127444 429124
153330 310448
7534 104027
108236 195845
4966 164942
21210 228006
155629 675479
41952 65412
22431 180380
128921 834344
119118 419950
75836 628554
61484 630962
68234 621983
128945 568779
131535 791382
109645 694915
34247 853562
6060 36367
77440 118688
145222 526763
14559 132436
20504 802925
26171 618775
35540 167845
135997 807022
78154 163254
37895 5415
110522 616453
103028 148487
147943 269618
82894 8464
4656 574375
115413 777394
90471 143020
11545 392047
125272 840170
42095 482100
8327 425637
26634 118290
100535 458626
122541 300093
133459 752665
136581 858446
113989 453884
78368 118897
30971 41309
108994 364815
119537 275177
107140 21107
111111 593641
51903 674759
3920 670549
110915 848956
70707 711735
89513 273069
33061 391145
67833 776822
110354 577311
4293 386112
88881 223759
116238 43504
151391 336315
43995 618902
114291 150371
136729 13112
128416 713174
48044 358968
106251 758222
99341 387698
109391 138410
68259 407576
153222 182177
149225 6040
94734 48332
78212 797441
32652 659512
141151 671378
63375 718725
27736 238362
52554 741169
94018 821328
19794 146634
6997 671986
62751 785120
116100 250338
60144 532948
131349 412178
43133 314171
131238 568476
51015 651905
124273 634560
86074 408987
115631 297924
141909 202731
85642 424012
17964 191735
3231 314737
71024 416669
43438 289035
153978 143958
28672 273807
155353 499879
114225 492842
139395 839249
67511 249701
72910 283751
156703 691458
87940 141736
40835 177302
3118 731796
36462 688626
85996 367784
32266 401322
154005 410977
147203 423277
28074 331128
81025 275396
48614 12851
59336 139774
30627 209401
25649 630392
73839 499450
46039 49055
80111 431984
121186 602000
19659 639508
41845 855003
146674 787987
92797 434185
2047 537048
114969 345783
76204 615331
98769 103945
20024 477629
62494 299732
2122 360447
12915 411442
123881 514966
155094 70457
106839 23314
154636 173637
44452 474445
7150 627607
21337 544101
150863 146319
149782 723903
144091 650429
12918 823737
97736 654136
98569 767927
65274 174300
110779 121396
134547 220088
47017 252007
103350 724408
153827 270635
92793 315994
70809 504882
105826 739216
42461 321702
3098 842754
148004 744408
151576 565480
74222 399151
47586 203295
35072 244047
90709 438388
85243 537577
23771 706957
6030 351711
47061 520951
74254 490101
46339 504490
122639 323449
65646 661424
1821 505184
17841 445925
53846 628059
151813 661922
115609 681157
55603 260383
12027 291563
57820 570633
157771 256124
7450 89606
92701 285160
18702 780737
72513 749659
3863 315980
139431 584012
48291 656669
63366 52145
47692 641263
20617 493897
120321 412105
115353 514553
134834 685174
67875 4586
18076 806866
59717 158398
16260 276505
106891 579204
86724 571225
72194 506944
14728 204072
130666 802281
109208 414617
73978 453341
130624 106535
53131 262551
60943 344326
134387 254224
93837 147055
58420 548013
93975 509724
50294 675549
50877 466696
104707 226241
19665 167784
83463 404563
23795 401958
70708 566143
12940 608520
98903 539255
53820 74228
18510 843230
90692 335494
131191 111380
16012 123071
146644 580017
97321 507858
87970 616573
118993 439247
154763 782197
31252 127448
151414 292081
150773 235307
115097 215890
142168 90891
5965 99134
111649 108362
39092 846058
131187 555698
59186 257936
151418 696478
93866 493422
59074 506403
15600 245986
60027 102351
43342 372029
13236 137672
81666 650736
76371 548846
36925 226280
158242 640683
113688 542748
109501 743225
77142 570189
147420 701411
70664 34312
131535 818453
27575 68000
142704 273846
50738 478710
30367 99208
4706 505448
4486 56102
132589 826650
64744 268872
150678 320546
91475 218752
67871 334296
80397 212641
32122 724148
125559 170348
82301 594131
115933 501552
116400 621223
131385 270459
18480 29095
141011 424591
68355 356897
24446 318677
149417 6165
131400 487640
38329 138681
126077 455299
102996 258186
37341 494274
40977 170603
151277 233304
74174 340170
114357 607221
56793 188012
123620 68609
137057 412115
90425 530144
142746 580065
56368 16619
74512 168398
157516 350056
138094 712839
22596 470468
12641 852281
64726 342985
90389 408147
67361 122318
123878 836699
99662 688042
32177 601726
88390 37169
74733 839863
93434 473001
116689 644441
155085 811598
52290 692764
146321 116651
135448 786791
111465 788374
34754 714908
25275 219291
22484 833480
48953 812066
62072 108705
144011 745544
147839 690480
22409 253875
46550 780731
143429 273017
74430 191627
97179 120770
11276 349121
129624 783719
3085 387672
125547 278713
108497 111606
124840 149175
22552 577509
123575 633640
20137 561250
20009 95525
120598 313857
70868 261835
18574 93067
107813 586882
81635 565043
68985 220365
74582 272181
150954 98650
135134 755651
139090 304464
56939 553841
155689 398437
