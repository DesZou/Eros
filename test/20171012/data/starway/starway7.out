7170.469196643
